slik_wrangler/preprocessing.py

```diff
@@ -161,18 +163,57 @@ def check_nan(dataframe=None, plot=False, display_inline=True):
 
     missing_percent = round((df['missing_counts'] / data.shape[0]) * 100, 1)
     df['missing_percent'] = missing_percent
-    nan_values = df.set_index('features')['missing_percent']
     
     
+    print_divider('Count and Percentage of missing value')
     if plot:
-        plot_nan(nan_values)
+        plot_nan(df,**kwargs)
     if display_inline:
-        print_divider('Count and Percentage of missing value')
-        display(df)
-    check_nan.df = df
+        return df
+    else:
+        check_nan.df = df
 
+def convert_dtype(dataframe:pd.DataFrame, columns: list, convert_type: str):
```

Comment:
New data type conversion function. Let me know what you guys think, especially edge cases. @Sensei-akin @ganiyuolalekan

Reply:
I decided to leave the conversion inplace because from my experience, people rarely do dtype conversion in another column. Might be wrong though.

Reply:
For the docstring, under convert_type, the kind of string expected should be specified i.e int, str and float. This way a user knows what string to pass. 

As regards using  "inplace", it could be a parameter, with a default value of True.

Reply:
I've checked out the function... it is good and the fact that it is inplace is actually perfect to me, but we might want to also leave that for the users to decide.

So for instance having an `inplace` parameter set to `True` by default and they can alternate if they liked.

Reply:
Sounds good. Thanks @Justus-coded @ganiyuolalekan 
I will modify based on your inputs.

Reply:
@AdesholaAfolabi think there are cases where this method will not work for float or int. I know I have faced sth similar. I had to exclude nans and try or i used the pandas.numeric method. 

possible error  Cannot convert non-finite values (NA or inf) to integer

Reply:
@AdesholaAfolabi https://stackoverflow.com/questions/48511484/data-type-conversion-error-valueerror-cannot-convert-non-finite-values-na-or

Reply:
@Sensei-akin yeah you're right. I'd include a param to ask the user to drop_nan or not before doing the conversion or I'd make the choice myself.